spreadsheets/collision_rank.yaml

```diff
@@ -108,6 +108,9 @@
 - {$layer: 'water', kind: 'sea'}
 - {$layer: 'pois', kind: 'nature_reserve'}
 - {$layer: 'water', kind: 'lake'}
+# for contours
+- _reserved: {count: 10}
+- {$layer: 'contours', kind: 'std'}
```

Comment:
kind should be `contour`

Reply:
I would put this at the very bottom of the file... just above the default value. 

These contour in this file are just for the label collisions, which should be least priority. If there are "index" contours (eg with a kind_detail) then those should have priority over generic contours.

Reply:
Updated according to your first comment.